facia-tool/public/js/modules/vars.js

```diff
@@ -1,5 +1,5 @@
 import CONST from 'constants/defaults';
-import isEqual from 'underscore';
+import _ from 'underscore';
```

Comment:
What is the thought behind this? Is it just that you are more aware below in the code that it is _score method instead of some own function?